gatsby-node.js

```diff
@@ -85,7 +100,7 @@ exports.createPages = async ({ graphql, actions }) => {
           summary
           body
           meta_description
-          published
+          published(formatString: "ddd DD MMM YYYY")
```

Comment:
@orlyohreally Let me know if this change resolves: https://github.com/ButterCMS/gatsby-applifting/issues/13 so I can close

Reply:
@ViolanteCodes yes, it resolves